sdks/java/extensions/sorter/src/main/java/org/apache/beam/sdk/extensions/sorter/SortValues.java

```diff
@@ -167,6 +170,55 @@ public void processElement(ProcessContext c) {
       }
     }
 
+    @FunctionalInterface
+    private interface ThrowingFunction<T, U> {
+      U apply(T t) throws IOException, CoderException;
+    }
+
+    private class KvByteTranslator {
```

Comment:
This feels like a particularly inelegant solution but I didn't have a better idea 😅